content/sdk/10.js/00.ethers/20.v6/00.getting-started.md

```diff
@@ -1,69 +1,134 @@
+
+```javascript
+  try {
+    const transactionHash = "0xaa065e5a57e1f8470a6f258e2b6eee87c547eab066b8620ce7f3fd51405665e1";
 ```
 
-Get the transaction by hash (helper function [toJSON](/sdk/js/ethers/v6/providers#tojson)):
+- Sets a constant `transactionHash` representing a specific transaction.
+
+### Fetching Transaction Details
+
+```javascript
+    try {
+      const result = await provider.getTransactionDetails(transactionHash);
+      if (result) {
+        console.log(`Transaction Details from zkSync: ${JSON.stringify(result)}`);
+        
+        console.log(`ethCommitTxHash: ${result.ethCommitTxHash}`);
+        console.log(`ethExecuteTxHash: ${result.ethExecuteTxHash}`);
+        console.log(`ethProveTxHash: ${result.ethProveTxHash}`);
+      } else {
+        console.log(`Transaction with hash ${transactionHash} not found in zkSync.`);
+      }
+    } catch (error) {
+      console.error(`Error fetching transaction details from zkSync: ${error}`);
+    }
+```
 
-```ts
-console.log(`Block: ${toJSON(await provider.getTransaction("0x9af27afed9a4dd018c0625ea1368afb8ba08e4cfb69b3e76dfb8521c8a87ecfc"))}`);
+- **provider.getTransactionDetails(transactionHash)**: Fetches details of the specified transaction.
+- If transaction details are found, logs the details and specific Ethereum proof hashes (ethCommitTxHash,
+ethExecuteTxHash, ethProveTxHash).
+- If no details are found, logs that the transaction was not found.
+- Catches and logs any errors encountered during fetching transaction details.
+
+### Error Handling
+
+```javascript
+  } catch (error) {
+    console.error(`Error: ${error}`);
+  }
+```
+
+- Catches and logs any errors encountered in the main try block.
+
+### Executing Main Function
+
+```javascript
+main()
+  .then(() => console.log("Script executed successfully"))
+  .catch((error) => console.error(`Error executing script: ${error}`));
 ```
 
-Also, the following examples demonstrate how to:
+- Executes the main function.
+- Logs a success message if the function executes without errors.
+- Logs any errors encountered during the execution of the main function.
+
+Run the script:
 
-1. [Deposit ETH and tokens from Ethereum into zkSync Era](https://github.com/zksync-sdk/zksync2-examples/blob/main/js/src/01_deposit.ts)
-2. [Transfer ETH and tokens on zkSync Era](https://github.com/zksync-sdk/zksync2-examples/blob/main/js/src/02_transfer.ts)
-3. [Withdraw ETH and tokens from zkSync Era to Ethereum](https://github.com/zksync-sdk/zksync2-examples/blob/main/js/src/04_withdraw.ts)
-4. [Use paymaster to pay fee with token](https://github.com/zksync-sdk/zksync2-examples/blob/main/js/src/22_use_paymaster.ts)
+```shell
+npx ts-node src/zksync.ts
+```
 
-Full code for all examples is available [here](https://github.com/zksync-sdk/zksync2-examples/tree/main/js/src).
-Examples are configured to interact with `zkSync Era` and `Sepolia` test networks.
+::div{class="text-center"}
+![js sdk ethers](/images/sdk/js-ethers.png){ width="50%"}
```

Comment:
We have a guideline to avoid using images for blocks of text. For logs like these, use the ```sh tag.